### DISCUSSION TITLE
V3 doc and list keys

### DISCUSSION THREAD

**Discussion by Fran-Rg** [etcd-io/etcd#15092] [Q&A] [answered]
Just created a cluster. `etcdctl get /` returns nothing whereas I've done before `etcdctl put hello world`

So how do I list?
I'm also confused as I can't find the HTTP API for /v3. This doc forces to use etcdtctl: https://etcd.io/docs/v3.5/dev-guide/interacting_v3/  and I don't understand how this should be read https://etcd.io/docs/v3.5/dev-guide/api_reference_v3/

I saw that previously there was a v2/keys endpoint, where is that now?

**Comment by SimFG** [comment, marked as answer]
you can try `etcdctl get --prefix=true ""`.
And I have developed [a tool](https://github.com/SimFG/etcd-analysis) about looking etcd data.
